OneCommunityGlobal/HighestGoodNetworkApp
# Jiangwei ceate unit test for TableFilter

# Description
Create unit tests for the TableFilter component to confirm whether it working as intended.

## Main changes explained:
- Create file TableFilter.test.js  to do the unit tests

## How to test:
1. check into the current branch
2. do npm install and run this PR locally
3. open file src → __tests__ → TableFilter.test.js
4. run `npm run test src/__tests__/TableFilter.test.js`


## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/56906163/8acddb6b-2a02-461f-86dd-3558cf2b2b0f)

## Note:
If all tests passed, everything worked as intended.
